serverless/messaging/api-cli/sqs-sns-aws-cli.mdx

```diff
@@ -86,7 +86,7 @@ The AWS-CLI is an open-source tool built on top of the AWS SDK for Python (Boto)
 
 ### Preparing and subscribing to an SQS target for SNS messages
 
-1. Use the following command to create a queue, which must be in the same namespace as the SNS topic:
+1. Use the following command to create a queue, which must be in the same Scaleway Project as the SNS topic:
```

Comment:
Is this correct?

Reply:
No, same. Only http(s) and lambda can work.